config/locales/en/file_attachments/edit.yml

```diff
@@ -5,3 +5,16 @@ en:
       attachment_title:
         heading: Edit title
         hint_text: Use the official title of the document
+      attachment_file:
+        heading: Replace file
+        description_govspeak: |
+          All files must be [accessible](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs){:target="_blank"}.
+
+          Files must be uploaded in an [open standards file format](https://www.gov.uk/guidance/content-design/planning-content#open-formats){:target="_blank"}.
+          For example, .odt not .docx.
+        filename_change_callout: |
```

Comment:
If we're rendering this as govspeak we should suffix the name with govspeak.

Reply:
Done 👍